OpenOversight/app/utils/constants.py

```diff
@@ -17,6 +18,7 @@
 KEY_ENV_DEV = "development"
 KEY_ENV_TESTING = "testing"
 KEY_ENV_PROD = "production"
+KEY_MAP_DATA = "MAP_DATA"
```

Comment:
Is this used anywhere?

Reply:
<img width="1029" alt="Screenshot 2025-03-26 at 5 37 59 PM" src="https://github.com/user-attachments/assets/3de22122-433e-4f9c-90bb-8899ab8af75a" />